kf_lib_data_ingest/common/io.py

```diff
@@ -62,7 +74,11 @@ def read_delimited_text_df(filepath_or_buffer, **kwargs):
     kwargs["engine"] = "python"
     kwargs["dtype"] = str
     kwargs["na_filter"] = False
-    return pandas.read_csv(__ascii_wrap(filepath_or_buffer), **kwargs)
+    kwargs["encoding"] = kwargs.get("encoding") or __detect_encoding(
```

Comment:
Hmm is this going to slow things down in a noticeable way for big dfs? Couple options.. 1) Could make the default behavior what we had before (no supplied encoding to read_df) and then add an `auto_detect_encoding` kwarg in read_df or 2) Always assume large text, detect encoding incrementally, stop when you find it: https://chardet.readthedocs.io/en/latest/usage.html?#advanced-usage

Reply:
It shouldn't. It only reads a 2kb from the file.

Reply:
Ohhh ok I was looking at chardet. I'll approve